### PR TITLE
New Templates

### PR DESCRIPTION
**Pull request by kraftp**

### CODE REVIEW

File: python/dbos-cron-template/app/main.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+app = FastAPI()
+DBOS(fastapi=app)
+counter = 0
+
+# This is a simple scheduled function.
+# It runs once a minute, incrementing the in-memory counter (the cron syntax "* * * * *" means "once a minute").
+# Learn more about scheduled functions here: https://docs.dbos.dev/python/tutorials/scheduled-workflows
+
+@DBOS.scheduled("* * * * *")
+@DBOS.workflow()
```
Comment (qianl15) [inline python/dbos-cron-template/app/main.py:16]: Make this a step because this is not deterministic

File: python/dbos-cron-template/README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+# Welcome to DBOS!
```
Comment (qianl15) [inline python/dbos-cron-template/README.md:1]: The title should be "DBOS Cron" or something more specific.

File: python/dbos-fastapi-template/README.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
+# Welcome to DBOS!
```
Comment (qianl15) [inline python/dbos-fastapi-template/README.md:1]: Maybe a more specific title like "DBOS FastAPI App"?

**Review by qianl15 — APPROVED** [review]

File: python/dbos-cron-template/README.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+This is a template application using DBOS to run some code on a (cron) schedule.
```
Comment (maxdml) [inline python/dbos-cron-template/README.md:3]:
```suggestion
This is a template application using DBOS to run code on a (cron) schedule.
```

File: python/dbos-cron-template/README.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+### Getting Started
+
+To get started building, edit this app's code in `app/main.py`.
```
Comment (maxdml) [inline python/dbos-cron-template/README.md:7]:
```suggestion
To get started building, edit `app/main.py`.
```

File: python/dbos-cron-template/README.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+You can use a DBOS Cloud database, a Docker container, or a local Postgres installation.
+
+<details>
+<summary><strong>Connecting to a DBOS Cloud Postgres database</strong></summary>
+
+First install the DBOS Cloud CLI (requires Node):
+
+```shell
+npm i -g @dbos-inc/dbos-cloud
+```
```
Comment (maxdml) [inline python/dbos-cron-template/README.md:35]: redundant with the previous instructions.

File: python/dbos-fastapi-template/README.md
Comment (maxdml) [inline python/dbos-fastapi-template/README.md:1]: same comments

**Pull request merged**